lib/navigation/preprocess_encoders.hpp

```diff
@@ -1,25 +1,42 @@
+#include "consts.hpp"
+
 #include <array>
 #include <cstdint>
 
 #include "core/types.hpp"
+#include "core/logger.hpp"
 
 namespace hyped::navigation {
 class EncodersPreprocessor {
```

Comment:
tidy up the ordering in this class a little

we usually go with public (constructors, methods, members) and then private (constructors, methods, members)